packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -284,215 +222,6 @@ QUnit.test('`if` helper with inline form: can use truthy param as binding', func
   equal(view.$().text(), 'yes');
 });
 
-QUnit.test('`if` helper with inline form: can use falsy param as binding', function() {
-  view = EmberView.create({
-    truthy: 'ok',
-    falsy: 'boom',
-    conditional: false,
-    template: compile('{{if view.conditional view.truthy view.falsy}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'boom');
-
-  run(function() {
-    view.set('falsy', 'no');
-  });
-
-  equal(view.$().text(), 'no');
-});
-
-QUnit.test('`if` helper with inline form: raises when using more than three arguments', function() {
```

Comment:
I've just realised that hasn't been moved over fully as the shared test uses the `@glimmer` prefix due to the assertion message being different. I'll update the shared assertion messages now so that we don't lose coverage